.github/workflows/e2e.yml

```diff
@@ -34,21 +34,17 @@ jobs:
             echo "deb [trusted=yes] https://apt.fury.io/kurtosis-tech/ /" | sudo tee /etc/apt/sources.list.d/kurtosis.list
             sudo apt update
             sudo apt install kurtosis-cli
-      - name: Clone optimism-package
-        run: |
-            git clone https://github.com/klkvr/optimism-package
-            cd optimism-package
-            git switch odyssey
       - name: Build Odyssey
         run: docker buildx build . --load -t ghcr.io/ithacaxyz/odyssey:latest
       - name: Run enclave
         id: kurtosis
         run: |
+            git clone https://github.com/klkvr/optimism-package
```

Comment:
why do we still need to use your optimism-package fork?

Reply:
once https://github.com/ethpandaops/optimism-package/pull/85 is merged there would be no need for it, can hold off this PR until it's merged

Reply:
ah got it, no need to wait- we can just change it later was just curious